src/pages/index.tsx

```diff
@@ -135,9 +132,10 @@ function HomepageDatabasesSection() {
         Databases
       </Heading>
       <div className={styles.body}>
-        Prisma ORM works seamlessly across most popular databases and service providers. <br />
+        Prisma ORM works seamlessly across most popular databases and service providers.
         Refer to our <Link to="/orm/reference/database-features">Database features matrix</Link> for information about supported features and types for
-        each database.
+        each database.<br />
+        Explore what databases and database providers <Link to="/accelerate/getting-started#prerequisites">Accelerate</Link> and <Link to="https://www.prisma.io/docs/pulse/database-setup">Pulse</Link> support. 
```

Comment:
This reads more like the inline links will just tell me about Accelerate and Pulse, vs. linking me to the pages they do. Can we rephrase to make that more clear?

Reply:
How about? 

To explore supported databases for Accelerate, visit the [prerequisites section] (/accelerate/getting-started#prerequisites). For Pulse, see the [database setup documentation](/pulse/database-setup).